Classes/Service/ConfigurationService.php

```diff
@@ -86,6 +81,10 @@ public function injectRecordService(WorkspacesAwareRecordService $recordService)
 	 */
 	public function __construct() {
 		$this->defaultIcon = '../' . ExtensionManagementUtility::siteRelPath('fluidcontent') . 'Resources/Public/Icons/Plugin.png';
+
+		$this->extConf = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['fluidcontent']);
+		$this->extConf['iconWidth'] = $this->extConf['iconWidth']?:'';
```

Comment:
There is only one other thing to remark: please preserve the constants and use those as default values for the alternative case in these ternary conditions. That will allow the site to show the default 24x24 sizes until the site administrator saves the EM config. E.g. `$this->extConf['iconWidth'] ? : self::ICON_WIDTH;` (note spaces for CGL compliance).